processor/src/chiplets/aux_trace/bus/kernel.rs

```diff
@@ -81,3 +131,46 @@ impl Display for KernelRomMessage {
         write!(f, "{{ proc digest: {:?} }}", self.kernel_proc_digest)
     }
 }
+
+/// A message linking unique kernel procedure hashes provided by public inputs, with hashes
+/// contained in the kernel ROM chiplet trace.
+pub struct KernelRomInitMessage {
+    pub kernel_proc_digest: [Felt; 4],
+}
+
+impl<E> BusMessage<E> for KernelRomInitMessage
+where
+    E: FieldElement<BaseField = Felt>,
+{
+    #[inline(always)]
+    fn value(&self, alphas: &[E]) -> E {
+        // In contrast to the responses to the decoder, we omit the label to simplify
+        // the verifier's work when initializing the virtual table's bus.
+        // This is safe, as long as all other messages are domain separated with a tag.
+        // TODO(adr1anh):
+        //   This may actually be unsafe, unless we explicitly use the 0 tag for
+        //   this message. The first element of a hash could correspond to one of the
+        //   the OP tags.
+        //   One solution could be to just use the 0 tag, and use challenges 2..5.
```

Comment:
As mentioned above, let's do it the right way, and if possible in this PR, as we will just have to live with it